src/frontend/src/common/components/library/data_subview/components/UsherTreeFlow/components/UsherConfirmationModal/index.tsx

```diff
@@ -34,11 +34,12 @@ const UsherConfirmationModal = ({
     <div>
       <RedirectConfirmationModal
         content={content}
-        img={usherLogo as string}
+        img={usherLogo as unknown as string}
```

Comment:
this was to address some typehint weirdness, it didn't like converting to a string directly.

Reply:
This shouldn't be necessary, and the original doesn't cause type failures for me. Is there an extra tool you have in your IDE?

Reply:
no i got the error from running make frontend-lint and i'm pretty sure that they were affecting the build, but only after i updated the dependencies. I can try removing it to see if i'm wrong

Reply:
so just tested it again and it wasn't breaking the build, this is what vs code was saying, not sure if it's best to cast to unknown or keep it as casting to string directly. I also tried using `String(usherLogo)` and that fixed the issue, what do you think is the best thing to do here? 

<img width="863" alt="Screen Shot 2021-11-03 at 3 56 54 PM" src="https://user-images.githubusercontent.com/13052132/140226313-1a3a84d6-fbb4-46ac-a747-21a8581682ae.png">

Reply:
i know String(usherLogo) is problematic if usherLogo is ever undefined